### ISSUE TITLE
Header for internal services

### ISSUE THREAD

**Issue by abbott567** [dwp/design-examples#32]
## Screenshot

![image](https://user-images.githubusercontent.com/9450064/45744839-5f995280-bbf7-11e8-9ca4-90a8937b6000.png)

## What problem does this design solve?
For internal services, we can't use the GOVUK crown logo.

## Which service are you researching this design on?
- Bereavement
- Move to Universal Credit

## How have you approached the design?
We originally replaced GOV.UK with DWP in the top left, but DWP staff know they work for DWP and a link to DWP's homepage provides no value.

**Comment by HelenOsg** [comment]
@abbott567 Is this ticket still relevant? Has there been further guidance on this since the ticket was created?

**Comment by abbott567** [comment]
@HelenOsg Yeah, it's still relevant. The crown should only be present when accompanied with the GOV.UK text, which is usually reserved for services hosted on service.gov.uk domains.
